python_modules/dagster-graphql/dagster_graphql/client/client.py

```diff
@@ -65,6 +69,8 @@ def __init__(
         port_number: Optional[int] = None,
         transport: Optional[Transport] = None,
         use_https: bool = False,
+        timeout: int = 60,
```

Comment:
this is a "breaking" change right? we previously imposed no timeout? For oss users hitting their own dagster-webservers they may have requests that take > 60 that they are ok with

Reply:
we may still choose to do it but i think it would be a breaking change changelog entry